### DISCUSSION TITLE
Network error with index error but chat bot working

### DISCUSSION THREAD

**Discussion by IHB4L** [logancyang/obsidian-copilot#1173] [Q&A]
i use claude-3-5-sonnet-20240620|3rd party (openai-format) for the chat ai and mistral-embed | 3rd party (openai-format) for the QA and the apis are working and the chat bot is working but the vault cannot be indexed and i get all the following errors
![382599820-fd451a1f-466d-4000-9023-16316b5e3ed4](https://github.com/user-attachments/assets/79edd4ed-07ad-4793-bd1f-368b7104a4e3)
![382599883-3be3d624-9a74-48ee-9662-9cd820008238](https://github.com/user-attachments/assets/a5cdff71-a1fd-481f-8398-0a7ed0dc512b)
![382599942-75b73316-0cf6-4769-a94a-f27130d315ca](https://github.com/user-attachments/assets/c8d5de5f-1254-42ab-bfdb-2c8f89f8475d)

**Comment by logancyang** [comment]
Is it related to #745

**Comment by IHB4L** [comment]
> Is it related to #745

I don't think so since the person uses llm studio, but it could be a similar issue.

**Comment by logancyang** [comment]
Need more info. Right now I can't reproduce this. Anybody who sees this as well please share your steps.
